.github/workflows/deploy-docs.yaml

```diff
@@ -10,6 +10,14 @@ jobs:
       contents: write
     runs-on: ubuntu-latest
     steps:
+    - uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # 5.1.0
+    - run: |
+        pushd nodeadm
+        GOOS=js GOARCH=wasm go build -o nodeadm.wasm cmd/nodeadm-wasm/main.go
+        popd
+        mkdir -p ./site/assets/wasm && cp ./nodeadm/nodeadm.wasm ./site/assets/wasm/
+        mkdir -p ./site/assets/javascripts && cp "$(go env GOROOT)/misc/wasm/wasm_exec.js" ./site/assets/javascripts/
```

Comment:
Pulling `wasm_exec.js` from the go installation is great!

Reply:
@ndbaker1 this failed: https://github.com/awslabs/amazon-eks-ami/actions/runs/12270078234/job/34234724232

You'll need to move this step after the `actions/checkout` one, that's where the code comes from

Reply:
aargh! how did i miss that one 😠 lets wrap this up: https://github.com/awslabs/amazon-eks-ami/pull/2088